src/Features/Core/Portable/GenerateFromMembers/GenerateFromMembersHelpers.cs

```diff
@@ -18,13 +17,9 @@
 
 namespace Microsoft.CodeAnalysis.GenerateFromMembers;
 
-internal abstract partial class AbstractGenerateFromMembersCodeRefactoringProvider : CodeRefactoringProvider
+internal static class GenerateFromMembersHelpers
```

Comment:
changed from an abstract base class to a static helper class.  this was just full of static helpers and no state.  so we didn't need inheritance.